webhookv3/webhook_event.go

```diff
@@ -0,0 +1,74 @@
+package webhookv3
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+
+	"github.com/PagerDuty/go-pagerduty"
+)
+
+type OutboundEventData struct {
+	Object map[string]interface{}
+	// The raw json data for use in structured unmarshalling.
+	RawData json.RawMessage
+}
+
+type OutboundEvent struct {
```

Comment:
Ah so the term "Outbound Event" is really only used in the [V3 Webhook documentation](https://developer.pagerduty.com/docs/ZG9jOjExMDI5NTkw-v3-overview). The reason for using this terminology is that in PagerDuty-Land, the term "Event" is synonymous with the Events sent into PagerDuty with the Events API.

We could make this just `Event` and `EventData` but I have found it worth being clear instead of concise here.

Reply:
We're going to get that already for free, as a side effect of how imports work within Go and the fact we have it in its own package. People who use it would need to it as `webhookv3.Event`. I'd agree if we decided to keep this in the primary `pagerduty` package.